proto/composerapis/astria/composer/v1alpha1/grpc_collector.proto

```diff
@@ -0,0 +1,18 @@
+syntax = 'proto3';
+
+package astria.composer.v1alpha1;
+
+import "google/protobuf/empty.proto";
+
+message RollupTx {
+  string rollup_id = 1;
+  bytes tx_bytes = 2;
+}
+
+message SubmitRollupTxsRequest {
+  repeated RollupTx rollup_txs = 1;
+}
+
+service GrpcCollectorService {
+  rpc SubmitRollupTxs(SubmitRollupTxsRequest) returns (google.protobuf.Empty) {}
```

Comment:
`google.protobuf.Empty` or `SubmitRollupTransactionsResponse` to add more fields in the future?

Reply:
@joroshiba could you let us know your thoughts on this?